README.md

```diff
@@ -2,6 +2,7 @@
 
 ### Building
 Make sure you have Jekyll and Bundle installed (see https://jekyllrb.com/docs/).
+Python 3.6+ is also required.

```

Comment:
if you want to build data gathered from the RIOT code base.

Reply:
The defaults are now also generated by the python script, so it's also required for building.